packages/api/src/providers/google-calendar/utils.ts

```diff
@@ -164,6 +186,52 @@ function parseGoogleCalendarAttendeeType(
   return "required";
 }
 
+function parseGoogleCalendarConferenceData(
+  conferenceData: GoogleCalendarEvent["conferenceData"],
+): CalendarEvent["conferenceData"] {
+  if (!conferenceData?.entryPoints?.length || !conferenceData.conferenceId) {
+    return undefined;
+  }
+
+  // Filter and map entry points to match our interface
+  const entryPoints = conferenceData.entryPoints
+    .filter(
+      (entry) =>
+        entry.entryPointType &&
+        entry.uri &&
+        (entry.entryPointType === "video" || entry.entryPointType === "phone"),
+    )
+    .map((entry) => ({
+      entryPointType: entry.entryPointType as "video" | "phone",
+      uri: entry.uri!,
+      label: entry.label ?? entry.uri!,
+      passcode:
+        entry.passcode ??
+        entry.password ??
+        entry.accessCode ??
+        entry.meetingCode ??
+        entry.pin ??
+        "",
+    }));
+
+  if (entryPoints.length === 0) {
+    return undefined;
+  }
+
+  return {
+    entryPoints: entryPoints.map((entry) => ({
+      entryPointType: entry.entryPointType,
+      uri: entry.uri,
+      meetingCode: entry.passcode,
+      password: entry.passcode,
+    })),
+    conferenceId: conferenceData.conferenceId,
+    conferenceSolution: {
+      name: conferenceData.conferenceSolution?.name || "Google Meet",
```

Comment:
is this always Google Meet according to the docs?

can't it be something else?